microcosm_sagemaker/testing/pipeline_harness.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+from inspect import signature
+
+
+class PipelineHarness(ABC):
+
+    """
+    Base class to support feature tests, by specifying the steps/pipes
+    of a pipeline
+    """
```

Comment:
nit: We use an empty line before end of docstring
```suggestion

    """
```